### PR TITLE
Make show_me! compatible with amazing_print + awesome_print

### LINKED ISSUE #31: replace awesome_print with amazing_print ?
@thbar Kiba is fantastic! Thanks for such a wonderful tool!

We're seeing this in our Ruby 2.7 bump branch: 

```
/.gem/ruby/2.7.1/gems/awesome_print-1.8.0/lib/awesome_print/formatters/base_formatter.rb:113: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/.gem/ruby/2.7.1/gems/awesome_print-1.8.0/lib/awesome_print/inspector.rb:63: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
```

AwesomePrint is no longer receiving active updates.  ~20mo ago now. 

[AmazingPrint](https://github.com/amazing-print/amazing_print#amazing-print) is a fork and actively managed alternative.

https://github.com/thbar/kiba-common/blob/72324e08a62d95c244d0966c21914f76a64d411f/lib/kiba-common/dsl_extensions/show_me.rb#L1

Thoughts?

### PR DESCRIPTION
**Pull request by joshRpowell**
@thbar closes #31

### CODE REVIEW

File: Changes.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 HEAD
 ----
 
+- Breaking: for Ruby 2.7+ `show_me!` requires `amazing_print` gem. `awesome_print` is no longer actively maintained.
```
Comment (joshRpowell) [inline Changes.md]: went with 2.7+ here until [awesome_print updates its .travis.yml](https://github.com/awesome-print/awesome_print/blob/4564fd74721562cbef2443f7d97109bf9192343d/.travis.yml#L8) for 2.7 when "either or" would hold true.

Comment (thbar) [inline Changes.md]:
```suggestion
- Breaking: `show_me!` is now compatible with both `awesome_print` and its modern replacement `amazing_print`. You will have to `require` the one you want to use in your code from now on.
```

**Review by thbar — CHANGES_REQUESTED** [review]
Thanks for the PR!

The tests are broken. Could you please require amazing print inside the "show me" test, so that the build comes back to green?

Thank you!

File: README.md [hunk 2]
```diff
@@ -277,7 +277,7 @@ logger = Logger.new(xxx)
 
 ### Kiba::Common::DSLExtensions::ShowMe
 
-A way to color-dump rows on the screen, useful during development when you are inspecting the data (requires the `awesome_print` gem).
+A way to color-dump rows on the screen, useful during development when you are inspecting the data (requires the `amazing_print` gem).
```
Comment (thbar) [inline README.md]:
```suggestion
A way to color-dump rows on the screen, useful during development when you are inspecting the data (requires either the `amazing_print` or the `awesome_print` gem).
```

File: README.md [hunk 3]
```diff
@@ -277,7 +277,7 @@ logger = Logger.new(xxx)
 
 ### Kiba::Common::DSLExtensions::ShowMe
 
-A way to color-dump rows on the screen, useful during development when you are inspecting the data (requires the `amazing_print` gem).
+A way to color-dump rows on the screen, useful during development when you are inspecting the data (requires either the `amazing_print` or the `awesome_print` gem).
```
Comment (joshRpowell) [inline README.md:280]: 👍

File: Changes.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 HEAD
 ----
 
-- Breaking: for Ruby 2.7+ `show_me!` requires `amazing_print` gem. `awesome_print` is no longer actively maintained.
+- Breaking: `show_me!` is now compatible with both `awesome_print` and its modern replacement `amazing_print`. You will have to `require` the one you want to use in your code from now on.
```
Comment (joshRpowell) [inline Changes.md:4]: 👍

**Comment by thbar** [comment]
Thank you @joshRpowell! I will merge now.

**Pull request merged**